showcase/build/iaf_cond_alpha_implicit.h

```diff
@@ -0,0 +1,888 @@
+  */
+  ~iaf_cond_alpha_implicit();
+  /**
+  * Import sets of overloaded virtual functions.
+  * This is necessary to ensure proper overload and overriding resolution.
+  * @see http://www.gotw.ca/gotw/005.htm.
+  */
+  using nest::Node::handles_test_event;
+  using nest::Node::handle;
+
+  /**
+  * Used to validate that we can send nest::SpikeEvent to desired target:port.
+  */
+  nest::port send_test_event(nest::Node &target, nest::rport receptor_type,
+                             nest::synindex, bool);
+
+  /**
+  * @defgroup mynest_handle Functions handling incoming events.
+  * We tell nest that we can handle incoming events of various types by
+  * defining @c handle() and @c connect_sender() for the given event.
+  * @{
+  */
+  void handle(nest::SpikeEvent &);         //! accept spikes
+  void handle(nest::CurrentEvent &);       //! accept input current
+  void handle(nest::DataLoggingRequest &); //! allow recording with multimeter
+
+  nest::port handles_test_event(nest::SpikeEvent &, nest::port);
+  nest::port handles_test_event(nest::CurrentEvent &, nest::port);
+  nest::port handles_test_event(nest::DataLoggingRequest &, nest::port);
+  /** @} */
+
+  // SLI communication functions:
+  void get_status(DictionaryDatum &) const;
+  void set_status(const DictionaryDatum &);
+
+private:
+  //! Reset parameters and state of neuron.
+
+  //! Reset state of neuron.
+  void init_state_(const Node &proto);
+
+  //! Reset internal buffers of neuron.
+  void init_buffers_();
+
+  //! Initialize auxiliary quantities, leave parameters and state untouched.
+  void calibrate();
+
+  //! Take neuron through given time interval
+  void update(nest::Time const &, const long, const long);
+
+  // The next two classes need to be friends to access the State_ class/member
+  friend class nest::RecordablesMap<iaf_cond_alpha_implicit>;
+  friend class nest::UniversalDataLogger<iaf_cond_alpha_implicit>;
+
+  /**
+  * Free parameters of the neuron.
+  *
+  *
+  *
+  * These are the parameters that can be set by the user through @c SetStatus.
+  * They are initialized from the model prototype when the node is created.
+  * Parameters do not change during calls to @c update() and are not reset by
+  * @c ResetNetwork.
+  *
+  * @note Parameters_ need neither copy constructor nor @c operator=(), since
+  *       all its members are copied properly by the default copy constructor
+  *       and assignment operator. Important:
+  *       - If Parameters_ contained @c Time members, you need to define the
+  *         assignment operator to recalibrate all members of type @c Time . You
+  *         may also want to define the assignment operator.
+  *       - If Parameters_ contained members that cannot copy themselves, such
+  *         as C-style arrays, you need to define the copy constructor and
+  *         assignment operator to copy those members.
+  */
+  struct Parameters_ {
+    /* generated by template org.nest.nestml.neuron.function.MemberDeclaration*/
```

Comment:
Is there a way to include this comment only once? It makes the code a lot harder to read and adds unnecessary lines to the code...